umbral-pre/src/capsule_frag.rs

```diff
@@ -50,14 +17,13 @@ pub struct CapsuleFragProof {
     kfrag_pok: CurvePoint,
     signature: CurveScalar,
     kfrag_signature: Signature,
-    metadata: HashedMetadata,
 }
 
 type PointSize = <CurvePoint as SerializableToArray>::Size;
 type ScalarSize = <CurveScalar as SerializableToArray>::Size;
 type SignatureSize = <Signature as SerializableToArray>::Size;
 type CapsuleFragProofSize =
-    op!(PointSize + PointSize + PointSize + PointSize + ScalarSize + SignatureSize + ScalarSize);
+    op!(PointSize + PointSize + PointSize + PointSize + ScalarSize + SignatureSize);
```

Comment:
Is this due to the metadata?

Reply:
Yep, it was kept hashed to scalar before.